crates/sui-types/src/bridge.rs

```diff
@@ -299,3 +317,8 @@ pub struct MoveTypeBridgeRecord {
     pub verified_signatures: Option<Vec<Vec<u8>>>,
     pub claimed: bool,
 }
+
+pub fn is_bridge_committee_initiated(object_store: &dyn ObjectStore) -> SuiResult<bool> {
```

Comment:
is this the function that should be called to stop the committee to be created multiple times?
this is not called at all right now?